### DISCUSSION TITLE
AOC 24B2W1 'does not support DDC/CI features'

### DISCUSSION THREAD

**Discussion by HStep20** [xanderfrangos/twinkle-tray#511] [Monitor Compatibility Issues]
According to twinkle tray, the monitor does not support DDC/CI features. Looking around in the monitor menu though, I see an option for DDC/CI features, which is currently set to enabled. Ive toggled it on and off to test, and both time it showed a similar error.

Oddly enough, I can't find the model number online at all, but instead a google search returns the '24B2XH' model, which looks like what I have. Perhaps TT is reporting the wrong model somehow, causing an issue with it

**Comment by xanderfrangos** [comment]
Twinkle Tray doesn't have a database of monitors, so it only reports what it gets back from Windows and the monitor. Check if reinstalling GPU drivers or using different ports (on both the GPU and monitor) helps. See the [troubleshooting page](https://github.com/xanderfrangos/twinkle-tray/wiki/Display-Detection-&-Support-Issues#common-reasons-for-monitors-not-being-supported-or-detected) for more info.
